slp_abacus/slp_abacus/validate/abacus_mapping_file_validator.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from slp_base import MultipleMappingFileValidator
+from slp_base.slp_base.schema import Schema
+
+logger = logging.getLogger(__name__)
+
+class AbacusMappingFileValidator(MultipleMappingFileValidator):
+    # schema_filename = 'abacus_mapping_schema.json'
+
+    def __init__(self, mappings_data: [bytes]):
+        super(AbacusMappingFileValidator, self).__init__(None, None)
+
+    def validate(self):
```

Comment:
By overwriting this method, we are preventing Abacus mapping files from being validated, which does not comply with our security guidelines. In this case, as we are going to use the same format for the mappings than in drawio, you can simply do the same as in `DrawioMappingFileValidator` without overwrite the `validate` method and copy-paste the `drawio_mapping_schema.json` (renaming it to `abacus_mapping_schema.json`) to the `slp_abacus` resources equivalent folder.

Reply:
got it, we'll ad it soon